src/content/docs/tutorial-errors/reduce-common-errors.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Reduce common errors
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+So far, you've learned how to scope out a service outage and get the error in front of the correct teams. Error analysis doesn't end after the solution is deployed, however. To return to our flooded yard metaphor, you've only diagnosed the leaking pipe as the direct cause of the flood, then ordered a patch to fix it. You aren't done, though. At this point, you should ask yourself: What caused the leaking pipe to begin with? 
+
+## Objectives [#objectives]
+
+This tutorial helps you understand the difference between direct cause and root cause so you can reduce common errors over time. By the end of this tutorial, you will be able to:
+
+* Route errors to the correct teams
+* Diagnose the root cause of an incident
+
+## Route errors to the correct teams [#route]
+
+To send errors to other teams, you need to return to the **Errors** summary page. If you don't own a component or functionality, you can use this page to send grouped errors to the correct teams.  
```

Comment:
You can assign an error group to another user from inside the detail view, you don't need to go back to the list view unless you want to bulk assign several error groups at the same time.